source/agora/consensus/data/Block.d

```diff
@@ -597,7 +569,7 @@ version (unittest)
         assert(revealed.length == key_pairs.length - missing_validators.length);
         try
         {
-            Hash random_seed = createRandomSeed(pre_images);
+            Hash random_seed = pre_images.reduce!((a, b) => hashMulti(a, b));
```

Comment:
In the future I would like to remove this duplication (we don't want it to get out of sync), but for the moment this is acceptable. The long term plan would be to make all tests that want to create a block to use `Ledger` so that we never create invalid blocks (because we need so much context to create a block nowadays).